File: java/PallindromeNumber.java
public class Algorithms {
static bool p(int n){
	if(n < 0)
		n=-n;
	
	if(n >= 0 && n < 10)
		return false;	

	int sum=0,rem=0,num=n;
	
	while(n > 0){
		rem = n%10;
		sum=(sum*10) + rem;
		n=n/10;
	}

	if(sum == num)
		return true;
	else
		return false;
}
  public static void main(String[] args) {
    int n = 2002;
	
	if(p(n))
		System.out.println("Yes, it's a pallindrome");
	else	
		System.out.println("Not a pallindrome");
  }
}

File: python/PallindromeNumber.py
def p(n):
	if n < 0:
		n=-n
	
	if n >= 0 and n < 10:
		return False	

	s,rem,num = 0,0,n
	
	while n > 0:
		rem = n%10
		s=(s*10) + rem
		n=n//10
	

	if s == num:
		return True
	else:
		return False
    
    
    
n = 2001
if p(n):
	print("Yes, it's a pallindrome")
else:
	print("Not a pallindrome")
